catroid/src/main/res/values/strings.xml

```diff
@@ -1078,12 +1076,12 @@
     <!-- Physics -->
     <string name="default_project_name_physics">My first physics program</string>
     <string name="brick_set_physics_object_type">Set motion type to</string>
```

Comment:
```suggestion
    <string name="brick_set_physics_object_type">Set your motion type to</string>
```
Makes it clear that it concerns this actor or object only, and more consistent with other brick names that order actors and objects to do certain things, e.g., `When you receive...` etc.